docs/video/obs/quicksync.md

```diff
@@ -3,4 +3,81 @@ description: OBS configuration for Intel GPUs
 icon: simple/intel
 ---
 
-# :simple-intel: QuickSync OBS Configuration
\ No newline at end of file
+<div class="annotate" markdown>
+
+# :simple-intel: QuickSync OBS Configuration
+
+Before following this, make sure that you've followed the [initial configuration](index.md) first. **Make sure that you test your settings!** See [Replay Buffer](index.md#replay-buffer).
+
+## :material-cog: Configuration
+:material-information-slab-circle: These settings are intended for recording Minecraft at 30-60 frames per second with acceptable quality and medium file sizes. If you're looking to record with high fps, look to [NVENC](#index.md#nvenc) as an alternative to QuickSync
```

Comment:
> If you're looking to record with high fps, look to [NVENC](#index.md#nvenc) as an alternative to QuickSync

instead of this, put a link to /video/obs/#where-to-go-from-here as there's the official encoder comparison

Reply:
resolved with 2a5ce56